python/TestHarness/suppressions/errors.supp

```diff
@@ -189,3 +189,50 @@
    fun:*boost*math*
    ...
 }
+{
+   meowbuild_noai6ai_cached
```

Comment:
I don't think you should name these new suppressions "meowbuild". Can you name them after the OS or some other more useful trait of the system?

Reply:
spack perhaps? All the files being scrutinized are originating from the the compiler stack supplied by a spack path.